browser/tor/tor_profile_service.cc

```diff
@@ -18,6 +16,11 @@
 #include "components/prefs/pref_registry_simple.h"
 #include "components/prefs/pref_service.h"
 
+#if BUILDFLAG(ENABLE_TOR)
```

Comment:
shouldn't the entire TorProfileService be left out of gn if tor is not enabled?

Reply:
fixed in 9bb8d68ed53a77079155beab160dea6807ee93b8